src/mudlet-lua/lua/KeyCodes.lua

```diff
@@ -0,0 +1,330 @@
+	["Kana_Lock"] = 0x0100112d,
+	["Kana_Shift"] = 0x0100112e,
+	["Eisu_Shift"] = 0x0100112f,
+	["Eisu_Toggle"] = 0x01001130,
+	["Hangul"] = 0x01001131,
+	["Hangul_Start"] = 0x01001132,
+	["Hangul_End"] = 0x01001133,
+	["Hangul_Hanja"] = 0x01001134,
+	["Hangul_Jamo"] = 0x01001135,
+	["Hangul_Romaja"] = 0x01001136,
+	["Hangul_Jeonja"] = 0x01001138,
+	["Hangul_Banja"] = 0x01001139,
+	["Hangul_PreHanja"] = 0x0100113a,
+	["Hangul_PostHanja"] = 0x0100113b,
+	["Hangul_Special"] = 0x0100113f,
+	["Dead_Grave"] = 0x01001250,
+	["Dead_Acute"] = 0x01001251,
+	["Dead_Circumflex"] = 0x01001252,
+	["Dead_Tilde"] = 0x01001253,
+	["Dead_Macron"] = 0x01001254,
+	["Dead_Breve"] = 0x01001255,
+	["Dead_Abovedot"] = 0x01001256,
+	["Dead_Diaeresis"] = 0x01001257,
+	["Dead_Abovering"] = 0x01001258,
+	["Dead_Doubleacute"] = 0x01001259,
+	["Dead_Caron"] = 0x0100125a,
+	["Dead_Cedilla"] = 0x0100125b,
+	["Dead_Ogonek"] = 0x0100125c,
+	["Dead_Iota"] = 0x0100125d,
+	["Dead_Voiced_Sound"] = 0x0100125e,
+	["Dead_Semivoiced_Sound"] = 0x0100125f,
+	["Dead_Belowdot"] = 0x01001260,
+	["Dead_Hook"] = 0x01001261,
+	["Dead_Horn"] = 0x01001262,
+	["Back"] = 0x01000061,
+	["Forward"] = 0x01000062,
+	["Stop"] = 0x01000063,
+	["Refresh"] = 0x01000064,
+	["VolumeDown"] = 0x01000070,
+	["VolumeMute"] = 0x01000071,
+	["VolumeUp"] = 0x01000072,
+	["BassBoost"] = 0x01000073,
+	["BassUp"] = 0x01000074,
+	["BassDown"] = 0x01000075,
+	["TrebleUp"] = 0x01000076,
+	["TrebleDown"] = 0x01000077,
+	["MediaPlay"] = 0x01000080,
+	["MediaStop"] = 0x01000081,
+	["MediaPrevious"] = 0x01000082,
+	["MediaNext"] = 0x01000083,
+	["MediaRecord"] = 0x01000084,
+	["HomePage"] = 0x01000090,
+	["Favorites"] = 0x01000091,
+	["Search"] = 0x01000092,
+	["Standby"] = 0x01000093,
+	["OpenUrl"] = 0x01000094,
+	["LaunchMail"] = 0x010000a0,
+	["LaunchMedia"] = 0x010000a1,
+	["Launch0"] = 0x010000a2,
+	["Launch1"] = 0x010000a3,
+	["Launch2"] = 0x010000a4,
+	["Launch3"] = 0x010000a5,
+	["Launch4"] = 0x010000a6,
+	["Launch5"] = 0x010000a7,
+	["Launch6"] = 0x010000a8,
+	["Launch7"] = 0x010000a9,
+	["Launch8"] = 0x010000aa,
+	["Launch9"] = 0x010000ab,
+	["LaunchA"] = 0x010000ac,
+	["LaunchB"] = 0x010000ad,
+	["LaunchC"] = 0x010000ae,
+	["LaunchD"] = 0x010000af,
+	["LaunchE"] = 0x010000b0,
+	["LaunchF"] = 0x010000b1,
+	["MediaLast"] = 0x0100ffff,
+	["unknown"] = 0x01ffffff,
+	["Call"] = 0x01100004,
+	["Context1"] = 0x01100000,
+	["Context2"] = 0x01100001,
+	["Context3"] = 0x01100002,
+	["Context4"] = 0x01100003,
+	["Flip"] = 0x01100006,
+	["Hangup"] = 0x01100005,
+	["No"] = 0x01010002,
+	["Select"] = 0x01010000,
+	["Yes"] = 0x01010001,
+	["Execute"] = 0x01020003,
+	["Printer"] = 0x01020002,
+	["Play"] = 0x01020005,
+	["Sleep"] = 0x01020004,
+	["Zoom"] = 0x01020006,
+	["Cancel"] = 0x01020001
+}
+
+mudlet.keymodifier = {
+  ["None"] = 0x00000000,
+  ["Shift"] = 0x02000000,
+  ["Control"] = 0x04000000,
+  ["Alt"] = 0x08000000,
+  ["Meta"] = 0x10000000,
+  ["Keypad"] = 0x20000000,
+  ["GroupSwitch"] = 0x40000000,  
+}
```

Comment:
Needs a new-line at the end of the file I think!

Also, if you are adding to the set of files that we need to distribute like this (and its contents __could__ instead be included inside one of the existing ones IMVHO) then please also add this file to the relevant list {the `LUA.files` one} in the main qmake project file (i.e.  `/src/src.pro`) as well otherwise it may get missed by one of the installers...